tensorboard/data/server/pip_package/build.py

```diff
@@ -37,18 +37,27 @@
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
+        "--out-dir",
+        help="Output directory into which to place the wheel; should already exist",
+        required=True,
+    )
+    binary_group = parser.add_mutually_exclusive_group(required=True)
+    binary_group.add_argument(
         "--server-binary",
         help="Path to the //tensorboard/data/server binary to bundle into the package",
     )
-    parser.add_argument(
-        "--out-dir",
-        help="Output directory into which to place the wheel; should already exist",
+    binary_group.add_argument(
+        "--universal",
+        help="Build a universal package with no server",
+        action="store_true",
```

Comment:
Is `store_true` a reserved keyword of setup tools or soemthing?

Reply:
`store_true` is `argparse`’s strange syntax for “this is a boolean flag
that does not take an argument; when the flag is passed, store `True` as
its value”. You can see it also in (e.g.) `--bind_all` in `core_plugin`.
It’s not related to `setuptools` specifically.